web/public/js/directives/cd-expanding-card/index.js

```diff
@@ -35,14 +35,18 @@
           //content.css('height', 'auto');
           content.css('height', contentWrapper.outerHeight() + 'px');
           $timeout(function () {
+            content.css('height', 'auto');
             ctrl.expanded = 'expanded';
           }, 300);
         } else if (force) {
```

Comment:
means you can't have both "collapsed" and "force"? 
In which case, 2 fn would be nicer?